data/data-pipeline/data_pipeline/utils.py

```diff
@@ -1409,6 +1409,8 @@ def get_excel_column_name(index: int) -> str:
         "ALI",
         "ALJ",
         "ALK",
+        "ALL",
```

Comment:
there's no real reason to add these -- i was testing something -- but no real reason to delete them either :)